debian/control

```diff
@@ -5,27 +5,11 @@ Maintainer: Wazo Maintainers <dev@wazo.community>
 Build-Depends:
  debhelper (>= 9),
  dh-python,
- python-all (>= 2.7),
- python-setuptools,
  python3-all,
  python3-setuptools
 Standards-Version: 3.9.6
-X-Python-Version: >= 2.7
 X-Python3-Version: >= 3.4
```

Comment:
Let's make it python 3.7 since we are using features that were not available in python 3.4 (f-strings)

Reply:
Oh yes, good point! I thought I had changed all of them, but I missed this one. Thanks!